tests/test_rtmidi.py

```diff
@@ -16,8 +16,8 @@ class RtMidiTestCase(unittest.TestCase):
     DELAY = 0.1
 
     def setUp(self):
-        self.midi_out = rtmidi.MidiOut(name=self.TEST_PORT_NAME)
-        self.midi_out.open_virtual_port()
+        self.midi_out = rtmidi.MidiOut()
+        self.midi_out.open_virtual_port(name=self.TEST_PORT_NAME)
```

Comment:
Of course, like you mentioned in thestk/rtmidi#54, the test does not work this way on ALSA. Admitted, it only worked before, because the way the ALSA API behaves and port ordering seems to be consistent, but I would rather leave this change out and wait for the upstream fix (or do the test setup differently).